stage1/init/init.go

```diff
@@ -228,6 +230,7 @@ func getArgsEnv(p *Pod, flavor string, systemdStage1Version string, debug bool)
 
 	switch flavor {
 	case "coreos":
+		args = append(args, filepath.Join(common.Stage1RootfsPath(p.Root), interpBin))
```

Comment:
Well if we don't depend on anything else on the host, why make an exception here? it's just another opportunity for a mismatching version right? (even if practically the chances of it causing issues might be low)